themes/openy_themes/openy_rose/templates/paragraph/paragraph--webform--default.html.twig

```diff
@@ -0,0 +1,57 @@
+{#
+/**
+ * @file
+ * Default theme implementation to display a paragraph.
+ *
+ * Available variables:
+ * - paragraph: Full paragraph entity.
+ *   - id: The paragraph ID.
+ *   - bundle: The type of the paragraph, for example, "image" or "text".
+ *   - authorid: The user ID of the paragraph author.
+ *   - createdtime: Formatted creation date. Preprocess functions can
+ *     reformat it by calling format_date() with the desired parameters on
+ *     $variables['paragraph']->getCreatedTime().
+ * - content: All paragraph items. Use {{ content }} to print them all,
+ *   or print a subset such as {{ content.field_example }}. Use
+ *   {{ content|without('field_example') }} to temporarily suppress the printing
+ *   of a given child element.
+ * - attributes: HTML attributes for the containing element.
+ *   The attributes.class element may contain one or more of the following
+ *   classes:
+ *   - paragraphs: The current template type (also known as a "theming hook").
+ *   - paragraphs--type-[type]: The current paragraphs type. For example, if the paragraph is an
+ *     "Image" it would result in "paragraphs--type--image". Note that the machine
+ *     name will often be in a short form of the human readable label.
+ *   - paragraphs--view-mode--[view_mode]: The View Mode of the paragraph; for example, a
+ *     preview would result in: "paragraphs--view-mode--preview", and
+ *     default: "paragraphs--view-mode--default".
+ * - view_mode: View mode; for example, "preview" or "full".
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_paragraph()
+ *
+ * @ingroup themeable
+ */
+#}
+{%
+set classes = [
+  'webform',
+  'container',
+  'clearfix',
+  'row',
+]
+%}
+<article{{ attributes.addClass(classes) }}>
+  <div class="col-sm-5 col-md-4 image">
+    <a href="{{ content.field_prgf_link.0['#url'] }}">{{ content.field_prgf_image }}</a>
+  </div>
+  <div class="col-sm-7 content">
+    <h2>{{ content.field_prgf_title }}</h2>
+    {{ content.field_prgf_description }}
+    <a href="{{ content.field_prgf_link.0['#url'] }}" class="btn btn-link blue">{{ content.field_prgf_link.0['#title'] }}</a>
+  </div>
+</article>
```

Comment:
Seems this template has been copied over, but wasn't adjusted for new paragraph.